meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -248,6 +248,13 @@ IMAGE_CMD_resin-sdcard () {
     # Finally add the fingerprint to vfat partition
     mcopy -i ${WORKDIR}/boot.img -v ${WORKDIR}/${RESIN_BOOT_FS_LABEL}.${FINGERPRINT_EXT} ::
 
+    # Generate and inject config.json
+    echo '{}' > ${WORKDIR}/config.json
+    if ${@bb.utils.contains('DISTRO_FEATURES','debug-image','true','false',d)}; then
```

Comment:
You should modify this by replacing `debug-image` with `development-image`. That PR is already up.